### PR TITLE
Enhance setup.py to include long_description

### PR DESCRIPTION
**Pull request by xmatthias**
## Summary

Adding `long_description` and `long_description_content_type` to setup.py, so once we publish to pypi it has the correct description. 

part of issue: #2112 

## Quick changelog

- Add long description (copy Readme.md if available).

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -36,6 +43,8 @@
 setup(name='freqtrade',
       version=__version__,
       description='Crypto Trading Bot',
+      long_description=readme_long,
+      long_description_content_type="text/markdown",
       url='https://github.com/freqtrade/freqtrade',
       author='gcarq and contributors',
       author_email='michael.egger@tsn.at',
```
Comment (hroff-1902) [inline setup.py]: This should probably be updated.

author='Freqtrade Team', relevant common email...

If @gcarq don't mind...

(May be done in a sep. PR)

**Comment by xmatthias** [comment]
> (Since it's impossible to see how this long_description will be rendered in pypi yet)

huh? check out this: https://pypi.org/project/freqtrade/2019.7.dev0/

it's not fully functional (and huge - due to test-data being included) - but you can see how it should look like.

**Comment by hroff-1902** [comment]
> > (Since it's impossible to see how this long_description will be rendered in pypi yet)
> 
> huh? check out this: https://pypi.org/project/freqtrade/2019.7.dev0/
> 
> it's not fully functional (and huge - due to test-data being included) - but you can see how it should look like.

Cool, seems I have missed this. 

LGTM 👍

**Comment by xmatthias** [comment]
it's only visible when looking at the pre-release branch ... ;)

File: setup.py (hunk 1 above)
Comment (gcarq) [reply setup.py]: Sure, go for it.

**Pull request merged**